### DISCUSSION TITLE
Wayland for Gnome?

### DISCUSSION THREAD

**Discussion by gitJWR** [SpiralLinux/SpiralLinux-project#157] [Q&A] [answered]
Hello,
I am running Spiral Linux (Gnome), and have moved to Sid-Unstable.  I noticed that it is running x11 by default.  Is Wayland also available as an option out of the box, or is that something I would need to install?  If installation needed, any guidance would be appreciated.  Thanks

**Comment by geckolinux** [comment, marked as answer]
Hi there, thanks for the question.
I personally don't like Wayland and it caused problems for VirtualBox users, so it's disabled with `WaylandEnable=false` in `/etc/gdm3/daemon.conf` . I believe if you revert that change it will work out of the box, but I haven't tested it.

**Reply by gitJWR** [reply]
Thanks for the quick reply, and I am very much enjoying the experience so far.  Great work!

**Reply by geckolinux** [reply]
Awesome, really glad to hear it! If you have any experiences good or bad in the future running Sid I'd be interested in hearing about it, as I've never really ventured down that road.

**Reply by gitJWR** [reply]
Will do!  I've had no trouble so far.
